### PR TITLE
Correct typo

### PR DESCRIPTION
**Pull request by salarson**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/enterprise/PortalLaunchScheduler.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ Formerly, portal launches could only be scheduled through SharePoint PowerShell.
 
 ### Launch portal with over 100k users
 
-If you are planning to migrate over 100TB, please submit a support request following the steps listed below. Make sure to include all requested information.
+If you are planning to launch are portal with over 100k users, please submit a support request following the steps listed below. Make sure to include all requested information.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/PortalLaunchScheduler.md]:
```suggestion
If you are planning to launch a portal with over 100,000 users, please submit a support request following the steps listed below. Make sure to include all requested information.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @salarson, please commit my suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks

**Pull request merged**